src/app/birth-tree/page.tsx

```diff
@@ -256,28 +256,63 @@ export default function BirthTree() {
                   aria-labelledby="child-modal-title"
                   aria-describedby="child-modal-description"
                 >
-                  <Box sx={{ ...style, width: 200 }}>
-                    <h2>{friend.name}さんへ</h2>
+                  <Box
+                    sx={{ ...style }}
+                    className="
+                      w-[350px] max-w-[90%] h-[350px]  max-h-[40%] 
+                      p-0 rounded-[20px] outline-none border-2
+                      border-mainpink sm:max-h-[50%]"
+                  >
                     <form
                       action="#"
                       onSubmit={(event) => {
                         saveMessage(event, friend.id);
                         handleClose();
                         handleSnackBarOpen();
                       }}
+                      className="h-[100%]"
                     >
-                      <textarea
-                        id={`message"-${friend.id}`}
-                        name="message"
-                        rows={5}
-                        title="modal-text-area"
-                        cols={15}
-                        style={{
-                          border: "2px solid #000",
-                        }}
-                      ></textarea>
-                      <Button onClick={handleClose}>閉じる</Button>
-                      <Button type="submit">提出</Button>
+                      <div className="pl-[5%] pr-[5%] h-[15%]">
+                        <div className="w-full flex items-center justify-between pt-[7px] border-b border-dashed border-mainpink">
+                          <Button disableRipple className="p-0 bg-[transparent]" onClick={handleClose}>
+                            <img
+                              src="/card-modal-back-button.svg"
+                              alt="backbutton"
+                            />
+                            <p className="text-textbrawn text-[14px] pl-[7px]">
```

Comment:
https://nerdcave.com/tailwind-cheat-sheet
tailwindにはこんな感じでサイズが用意されてるから、使える場合はそっちを使おう！

```suggestion
                            <p className="text-textbrawn text-sm pl-[7px]">
```

`pt-[7px]`とかの細かいデザインは俺も見習います🙇